src/websocket-events/websocket-events.base.ts

```diff
@@ -58,6 +58,10 @@ export interface WebSocketStart extends WebSocketActionBase {
   namespace: string;
```

Comment:
Can probably delete the prior `WebSocketMessage` type above if we're not using it anywhere.

Reply:
This is still used for processing things that come from EVMConnect